code/script.js

```diff
@@ -225,101 +232,173 @@ const generateBoard = () => {
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log(secret); //Writes out the randomly selected secret person in the console for test
```

Comment:
You can delete this console so the user would not find out about the "secret person" in Console ;)